_layouts/projects.html

```diff
@@ -27,60 +27,42 @@
             endfor %}
      ];
     </script>
-    <script type="text/javascript" src="./data/admins.js"></script>
-    <script type="text/javascript" src="./vendors/angular-sanitize/angular-sanitize.js"></script>
-    <script type="text/javascript" src="./vendors/showdown/showdown.js"></script>
-    <script type="text/javascript" src="./vendors/markdown/markdown.js"></script>
-    <script type="text/javascript" src="./resources/js/app.js"></script>
-
-    <link rel="stylesheet" type="text/css" href="./vendors/bootstrap/css/bootstrap.min.css">
-    <link rel="stylesheet" type="text/css" href="./vendors/materialize/css/materialize.css">
-    <link rel="stylesheet" type="text/css" href="./resources/css/style.css">
-
-    <link href="https://fonts.googleapis.com/css?family=Roboto+Mono:300,300i,400,400i,700,700i|Roboto:100,200,300,400,500,600,700,800,900|Ubuntu+Mono|Overpass+Mono|Inconsolata" rel="stylesheet">
```

Comment:
I think this is part of the problem, this stylesheet needs to be in the page somewhere to download roboto if needed.